go/worker/compute/executor/committee/node.go

```diff
@@ -1412,7 +1467,10 @@ func NewNode(
 	roleProvider registration.RoleProvider,
 	scheduleCheckTxEnabled bool,
 	scheduleMaxQueueSize uint64,
+	lastScheduledCacheSize uint64,
 ) (*Node, error) {
+	var err error
```

Comment:
Why is this needed, isn't `err` defined when `NewRuntimeHostNode` is called below?

Reply:
You are right, thx :+1: